code-challenges.test.js

```diff
@@ -15,31 +15,63 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// Pseudo code:
+// Create a function takes an array and remove the first item from the array and shuffles remaining content
+// Use the following mutators:
+//    .shift - to remove the zeroth index ot the array
+//    .shuffle() method - to shuffle array
 
 
+describe("hue", () => {
+    var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+
+    it("removes the first item from the array and shuffles the remaining content", () => {
+        expect(hue(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]));
+
+        expect(hue(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre",]));
+    })
+})
 
 // b) Create the function that makes the test pass.
 
+const hue = (arr) => {
+    arr.shift()
+    return arr.sort((a, b) => 0.5 - Math.random())
+}
```

Comment:
Nice work. Very concise. In this case since you are not using the `a` and `b` in the sort logic, you can pass an empty set of parentheses.